### PR TITLE
New Mod : Obscure Military Armory

### PR DESCRIPTION
**Pull request by RickyVanz**
#### Summary
```SUMMARY: Mods "Adding new mod : Obscure Military Armory."```

#### Purpose of change
to add more mods variety to the game.

#### Describe the solution
This mod adds new, unique weapons for military plus melee weapons that has been suggested by Community. most of the weapons is designed to be powerful and intended for high-difficulty risk.

#### Describe alternatives you've considered
any feedback or suggestion is always accepted, as there may be something missed by me.

#### Testing
i've been testing this mod in-game to find mistakes like missing separator or mislocated commas, i also test the weapons and melee to make sure it work perfectly.

#### Additional context
making a mod based on Community suggestion is much more fun than making a mod using only sole idea from internet, as Community suggestion sometimes can make sense for apocalyptic theme of C:DDA.

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
We already have one "more guns" mod - Icecoon's Arsenal. Consider adding your weapons to existing mod instead of creating a new one with similar theme.
Also, lint you changes.

**Comment by RickyVanz** [comment]
so, should i put my weapons into Icecoon's Arsenal mod?

**Comment by Maddremor** [comment]
I've been thinking of the benefits of having a mod for esoteric guns that wouldn't fit in mainline, so this is welcome. However, mixing guns existing IRL and purely fictional and sci-fi weaponry seems unnecessary since they have different themes.

File: data/mods/ObscureArmory/gun.json [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+	"burst": 5,
+	"loudness": 20,
+	"flags": [ "NEVER_JAMS" ],
+	"magazines": [ [ "shot", [ "pancormag" ] ] ]
+  },
+  {
+    "id": "armgun",
+	"type": "GUN",
+	"name": "Mk-1 Arm Gun",
+	"description": "Prototype Arm-mounted gun, chambered in 9mm and uses muscle sensor as trigger, this weapon is designed by somebody who thinks why punch zombies when you can shoot a bullet from your arm.",
```
Comment (Qrox) [inline data/mods/ObscureArmory/gun.json]:
```suggestion
	"description": "Prototype arm-mounted gun, chambered in 9mm and uses muscle sensor as trigger, this weapon is designed by somebody who thinks why punch zombies when you can shoot a bullet from your arm.",
```

**Comment by RickyVanz** [comment]
well during the making of this mod, i am confused about which theme should i pick (i was planning this mod to be fully IRL weapons) and ended up making this mix of fictional and IRL mod.

**Comment by Maddremor** [comment]
My suggestion would be to make two mods, and then split up Icecoon's (with attribution added to the newly created mods) according to weapon type. Also create a third mod (or add to the medieval mod) pre modern melee weapons.

Then again, I have no authority on this project, so I hold no more truth than anyone else.

**Comment by RickyVanz** [comment]
that's nice idea.. very well, i'll close this PR and make new one.

**Comment by kevingranade** [comment]
Just to reinforce, I agree with @Maddremor's suggestion.

**Pull request closed without merging**